q2_demux/_demux.py

```diff
@@ -329,7 +329,9 @@ def emp_single(seqs: BarcodeSequenceFastqIterator,
         raise ValueError('No sequences were mapped to samples. Check that '
                          'your barcodes are in the correct orientation (see '
                          'the rev_comp_barcodes and/or '
-                         'rev_comp_mapping_barcodes options).')
+                         'rev_comp_mapping_barcodes options). If barcodes are '
+                         'NOT Golay format provide no_golay_error_correction '
```

Comment:
The parameter should be listed as `golay_error_correction`, the `no` prefix is added by the CLI, but isn't in all interfaces. I believe q2cli will also underline the text if it matches the parameter name exactly.

This means the error needs to be slightly reworded to say something about setting it to False.

Reply:
I see, I will make those changes and get pushed back up ASAP 👍